docker/Dockerfile

```diff
@@ -14,13 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE
+ARG BASE_PYTHON_VERSION
```

Comment:
This arg is an image, not just a version number, what about `BASE_PYTHON_IMAGE`?

Reply:
Makes sense!